mne/event.py

```diff
@@ -520,9 +523,9 @@ def _find_events(
             events = np.insert(events, 0, [first_samp, 0, initial_value], axis=0)
         else:
             logger.info(
-                "Trigger channel has a non-zero initial value of {} "
-                "(consider using initial_event=True to detect this "
-                "event)".format(initial_value)
+                f"Trigger channel {ch_name} has a non-zero initial value of "
+                "{initial_value} (consider using initial_event=True to detect this "
```

Comment:
While analyzing this dataset I initially tried `mne.find_events` on it and it wasn't immediately obvious to me what stim channel was actually being used, so I improved `mne.find_events` to log what it's actually using as the event channel.